invenio_app_rdm/records_ui/templates/semantic-ui/invenio_app_rdm/records/dashboard.html

```diff
@@ -15,16 +15,22 @@
 {%- endblock javascript %}
 
 {%- block page_body %}
-<div class="ui container dashboard-user-header-container">
-  <div class="ui items">
-    <div class="item">
-      <div class="ui tiny circular image">
-        <img src="/static/images/placeholder.png" />
-      </div>
-      <div class="content">
-        <span class="header">{{current_user.email}}</span>
-        <div class="meta">
-          <span>European Organization of Nuclear Research (CERN)</span>
+<div class="dashboard-header">
+  <div class="ui container">
+    <div class="ui items">
+      <div class="item">
+        <div class="ui image tinier circular circle">
+          <img src="/static/images/placeholder.png" />
+        </div>
+        <div class="content">
+          {% if username %}
+            <span class="header">{{username}}</span>
+          {% else %}
+            <span class="header">{{current_user.email}}</span>
+          {% endif %}
+          <div class="meta">
+            <span>Unknown organization</span>
```

Comment:
Please remove it, it will be ugly. Check how it looks without. We mention this in the stand up and we said that it won't be nice at all.
```suggestion

```